src/main/resources/db/migration/V7__EventToSendInsertionTime.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE event_to_send ADD column insertion_time TIMESTAMP NOT NULL DEFAULT '2021-01-01 00:00:00.000';
+ALTER TABLE event_to_send ALTER COLUMN insertion_time DROP DEFAULT;
```

Comment:
please add newline here

Reply:
I'm sure there is a reason, but why does the first line create with a default, only for the 2nd line to remove the default?

Reply:
The default is there because you can't add a NOT NULL column to an existing table without also populating that column for the existing rows (even tho there won't be any in this table)